mmcv/runner/hooks/logger/text.py

```diff
@@ -14,6 +15,12 @@ def log(self, runner):
             log_str = 'Epoch({}) [{}][{}]\t'.format(runner.mode, runner.epoch,
                                                     runner.inner_iter + 1)
         if 'time' in runner.log_buffer.output:
+            tot_time_sec = runner.log_buffer.output['time'] + \
+                runner.log_buffer.output['data_time']
+            eta_sec = tot_time_sec * \
+                (len(runner.data_loader) * runner.max_epochs - runner._iter)
```

Comment:
There are some indicators that can be used to compute the remaining iters.
- runner.max_iters
- runner.iter